master/getting-started/mesos/vagrant/Vagrantfile

```diff
@@ -177,7 +174,7 @@ Vagrant.configure("2") do |config|
       host.vm.provision :shell, inline: "chmod +x /usr/bin/calicoctl"
 
       # Install the Calico service files and start the Calico services.
-      ["calico", "calico-libnetwork"].each do |service_name|
+      ["calico",].each do |service_name|
```

Comment:
Can you de-loop this and just statically run the commands with service_name replaced with "calico"?

Also, delete the calico-libnetwork.service: https://github.com/projectcalico/calico/blob/master/master/getting-started/mesos/vagrant/units/calico-libnetwork.service